project/ISSUE-TRIAGE.md

```diff
@@ -31,53 +28,37 @@ If the author provides the standard information but you are still unable to tria
-applied to the issue or pull request using labels.  You can apply one or more labels.
-
-
-Kinds of classifications:
+#### Issue kind
 
 | Kind             | Description                                                                                                                     |
 |------------------|---------------------------------------------------------------------------------------------------------------------------------|
-| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
-| kind/cleanup     | Refactoring code or otherwise clarifying documentation.                                                                         |
-| kind/content     | Content that is not documentation such as help or error messages.                                                               |
-| kind/graphics    | Work involving graphics skill                                                                                                   |
-| kind/regression  | Regressions are usually easy fixes as hopefully the action worked previously and git history can be used to propose a solution. |
 | kind/bug         | Bugs are bugs. The cause may or may not be known at triage time so debugging should be taken account into the time estimate.    |
+| kind/docs        | Writing documentation, man pages, articles, blogs, or other significant word-driven task.                                       |
+| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
 | kind/feature     | Functionality or other elements that the project does not currently support.  Features are new and shinny.                      |
 | kind/question    | Contains a user or contributor question requiring a response.                                                                   |
-| kind/usecase     | A description of a user or contributor situation requiring a response perhaps in code or documentation.                         |
-| kind/writing     | Writing documentation, man pages, articles, blogs, or other significant word-driven task.                                       |
-| kind/test        | Tests or test infrastructure needs adding or updating.                                                                                                 |
 
+#### Functional area
 
-Contributors can add labels by using a `+kind/bug` in an issue or pull request comment.  
+| Area                      |
+|---------------------------|
+| area/api                  |
+| area/builder              |
+| area/cli                  |
+| area/kernel               |
+| area/runtime              |
+| area/storage              |
+| area/storage/aufs         |
+| area/storage/btrfs        |
+| area/storage/devicemapper |
+| area/storage/overlay      |
+| area/storage/zfs          |
```

Comment:
think the current idea was `kind/docs` + `area/what's-it-about`; https://github.com/docker/docker/pull/14719#discussion_r35476239

both approaches make sense in a way, I think we should just try if it works or not